src/flecs_private.h

```diff
@@ -425,17 +425,22 @@ ecs_row_t ecs_to_row(
 uint64_t ecs_from_row(
     ecs_row_t row);
 
+/* Utility that build a descriptive error string*/
+void build_error_string(char *error, const char* signature, long position, const char *system_id);
+
 /* Utility that parses system signature */
 int ecs_parse_component_expr(
     ecs_world_t *world,
     const char *sig,
     ecs_parse_action_t action,
-    void *ctx);
+    void *ctx,
+    const char *system_id);
```

Comment:
This is a bit nitpicking- but I usually keep a `ctx` argument as last argument of a function. I'd change it to the system entity handle, and move it to one spot before `ctx`.

Reply:
done.